### ISSUE TITLE
[confirm] function, Whether I enter y or n, it gives false

### ISSUE THREAD

**Issue by MrCarbone** [oven-sh/bun#10375]
### What version of Bun is running?

1.1.4

### What platform is your computer?

windows

### What steps can reproduce the bug?

[confirm] function, Whether I enter y or n, it gives false

**Comment by Electroid** [comment]
Hi, we don't have enough information to fix the issue you're seeing. Please provide a package.json, code snippet, and the commands you ran.

**Comment by Jarred-Sumner** [comment]
This may have been fixed as of Bun v1.1.12
```
PS C:\bun> bun --print 'confirm("YES/NO?")'
YES/NO? [y/N] y
true
PS C:\bun> bun --print 'confirm("YES/NO?")'
YES/NO? [y/N] n
false
PS C:\bun> bun --print 'confirm("YES/NO?")'
YES/NO? [y/N] N
false
PS C:\bun> bun --print 'confirm("YES/NO?")'
YES/NO? [y/N] Y
true
PS C:\bun> bun --print 'confirm("YES/NO?")'
YES/NO? [y/N]
false
PS C:\bun>
```
